atomic_reactor/plugins/prepub_squash.py

```diff
@@ -64,7 +62,7 @@ def __init__(self, tasker, workflow, tag=None, from_base=True, from_layer=None,
         :param from_base: bool, squash from base-image layer, on by default
         :param from_layer: layer from we will squash - if specified, takes precedence over from_base
         :param tag: str, new name of the image - by default use the former one
-        :param remove_former_image: bool, remove unsquashed image?
+        :param remove_former_image: not used
```

Comment:
Can this actually be removed from the constructor? `atomic-reactor` should ignore unknown args: https://github.com/projectatomic/atomic-reactor/blob/b14b5abd75e2b1cb67be6e39fafcf2b98e5a8bbe/atomic_reactor/plugin.py#L326